src/shared/components/hero-section/index.js

```diff
@@ -137,20 +119,17 @@ class Hero extends Component {
   }
 
   handleAxiosError = (error) => {
-    let errorMessage = 'Something went wrong while fetching package data: '
+    const { messages } = this.props.intl
+
```

Comment:
Perhaps we should `console.error(error)` so that the real error gets logged in the console?